src/nvidia-cuda/install.sh

```diff
@@ -55,23 +58,33 @@ fi
 echo "Installing CUDA libraries..."
 apt-get install -yq "$cuda_pkg"
 
-if [ "$INSTALL_CUDNN" = "true" ]; then
+if [ "$INSTALL_CUDNN" = "true" ] || [ "$INSTALL_CUDNNDEV" = "true" ]; then
     # Ensure that the requested version of cuDNN is available AND compatible
     cudnn_pkg_version="libcudnn8=${CUDNN_VERSION}-1+cuda${CUDA_VERSION}"
     if ! apt-cache show "$cudnn_pkg_version"; then
         echo "The requested version of cuDNN is not available: cuDNN $CUDNN_VERSION for CUDA $CUDA_VERSION"
```

Comment:
Let's say `INSTALL_CUDNN` is false, but `INSTALL_CUDNNDEV` is true, and there's case of unavailability, then the Feature will fail with this error. This seems completely unexpected, instead, can we move changes for installing cuDNN DEV to an isolated condition?

Reply:
I made the installation of cudnn-dev isolated.